tools/testoperator/src/tests/bench/mod.rs

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2024-2025 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use super::get_app_and_start_request;
+use crate::commands::TestArgs;
+use std::time::Duration;
+use test_framework::{
+    anyhow,
+    queries::{QueryOverrides, QuerySet},
+    spiced::SpicedInstance,
+    spicetest::{EndCondition, SpiceTest},
+};
+
+pub(crate) fn export(args: &TestArgs) -> anyhow::Result<()> {
+    let (_, mut start_request) = get_app_and_start_request(args)?;
+
+    start_request.prepare()?;
+    let tempdir_path = start_request.get_tempdir_path();
+
+    println!(
+        "Exported spicepod environment to: {}",
+        tempdir_path.to_string_lossy()
+    );
+
+    Ok(())
+}
+
+pub(crate) async fn run(args: &TestArgs) -> anyhow::Result<()> {
+    let query_set = QuerySet::from(args.query_set.clone());
+    let query_overrides = args.query_overrides.clone().map(QueryOverrides::from);
+    let queries = query_set.get_queries(query_overrides);
+
+    let (app, start_request) = get_app_and_start_request(args)?;
+    let mut spiced_instance = SpicedInstance::start(start_request).await?;
+
+    spiced_instance
+        .wait_for_ready(Duration::from_secs(10))
```

Comment:
```suggestion
        .wait_for_ready(Duration::from_secs(args.ready_wait.unwrap_or(30) as u64))
```

There is a parameter here we should use.